site/docs/transform.md

```diff
@@ -51,25 +51,44 @@ This example use `calculate` to derive a new field, then `filter` data based on
 
 ### Filter
 
-Vega-Lite's `transform.filter` property can be (1) a filter predicate object, (2) [Vega Expression](https://github.com/vega/vega/wiki/Expressions) string or (3) an array of filter predicates (either predicate object or expression string) that must be all true for a datum to be include.  
+Vega-Lite's `transform.filter` property can be (1) a filter predicate object, (2) [Vega Expression](https://github.com/vega/vega/wiki/Expressions) string or (3) an array of filter predicates (either predicate object or expression string) that must be all true for a datum to be include.
 
 #### Filter Object
 
-For a filter object, a `field` must be provided with one of the filter operators (`equal`, `in`, `range`).  The following table describes each of these properties.
+For a filter object, a `field` must be provided with one of the filter operators (`equal`, `in`, `range`).  Values of these operators can be primitive types (string, number, boolean) or a [DateTime definition object](#datetime) for describiing time. In addition, `timeUnit` can be provided to further transform a temporal `field`.
+
+The following table describes properties of a filter object.
 
 | Property      | Type          | Description    |
 | :------------ |:-------------:| :------------- |
 | field         | String        | Field to be filtered. |
-| equal         | String | Number | Boolean | Value that the `field`'s value should be equal to. |
-| range          | Number[]      | Array of length describing (inclusive) minimum and maximum values for the `field`'s value to be included in the filtered data. |
-| in         | Array         | A set of values that the `field`'s value should be a member of, for a data item included in the filtered data. |
+| equal         | String | Number | DateTime | Boolean | Value that the `field`'s value should be equal to. |
+| range          | Number[] | DateTime[] | Array of length describing (inclusive) minimum and maximum values for the `field`'s value to be included in the filtered data. |
+| in         | String[] | Number[] | DateTime[] | A set of values that the `field`'s value should be a member of, for a data item included in the filtered data. |
+
+{:#datetime}
```

Comment:
`{:#datetime}` seems not working. Maybe we need to use just a longer version.

**Suggestion**
Remove `{:#datetime}`
Change `[DateTime definition object](#datetime)` to `[DateTime definition object](#date-time-definition-object)`